build_projects/shared-build-targets-utils/Utils/HostVersion.cs

```diff
@@ -64,9 +64,9 @@ public override string ToString()
         public string LatestHostBuildMajor => CommitCountString;
         public string LatestHostBuildMinor => "00";
         public bool EnsureStableVersion => false;
-        public VerInfo LatestHostVersion => new VerInfo(1, 0, 2, ReleaseSuffix, LatestHostBuildMajor, LatestHostBuildMinor);
-        public VerInfo LatestHostFxrVersion => new VerInfo(1, 0, 2, ReleaseSuffix, LatestHostBuildMajor, LatestHostBuildMinor);
-        public VerInfo LatestHostPolicyVersion => new VerInfo(1, 0, 2, ReleaseSuffix, LatestHostBuildMajor, LatestHostBuildMinor);
+        public VerInfo LatestHostVersion => new VerInfo(1, 0, 2, ReleaseSuffix, LatestHostBuildMajor, LatestHostBuildMinor, CommitCountString);
```

Comment:
This constructor is pretty confusing. I don't understand why we have commit count twice, but don't block the merge on it.